packages/data/src/property-table.ts

```
@@ -292,7 +292,8 @@ function getPropertyValue(table: PropertyTable, idx: number, strings: StringTabl
     case PropertyValueType.Identifier:
     case PropertyValueType.Text:
     case PropertyValueType.Enum:
-      return table.valueString[idx] >= 0 ? strings.get(table.valueString[idx]) : null;
+      const si = table.valueString[idx];
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)